CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contributing
+
+First of all, thank you for contributing to the Software Gardening Almanac! 🎉 💯 🪴
+We're absolutely grateful for the kindness of your effort applied to grow this project into the best that it can be.
+
+This document contains guidelines on how to most effectively contribute to this project.
+
+If you are stuck or would like clarifications, please feel free to ask any questions or for help.
+
+## Code of conduct
+
+This project is governed by our [Code of Conduct (CoC) policy](https://github.com/software-gardening/almanac?tab=coc-ov-file) (located [here](https://github.com/software-gardening/.github/blob/main/CODE_OF_CONDUCT.md)).
+By participating, you are expected to uphold this code.
+Please report unacceptable behavior by following the procedures listed under the [CoC Enforcement section](https://github.com/software-gardening/almanac?tab=coc-ov-file#enforcement).
+
+## Security
+
+Please see our [Security policy](https://github.com/software-gardening/almanac?tab=security-ov-file) (located [here](https://github.com/software-gardening/.github/blob/main/SECURITY.md)) for more information on security practices and recommendations associated with this project.
+
+## Quick links
+
+- Documentation: <https://github.com/software-gardening/almanac>
+- Issue tracker: <https://github.com/software-gardening/almanac/issues>
+
+## Process
+
+### Reporting bugs or suggesting enhancements
+
+We’re deeply committed to a smooth and intuitive user experience which helps people benefit from the content found within this project.
+This commitment requires a good relationship and open communication with our users.
+
+We encourage you to report bugs or propose enhancements to improve the Software Gardening Almanac as a [GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) associated with the repository.
```

Comment:
```suggestion
We encourage you to file a [GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) to report bugs or propose enhancements to improve the Software Gardening Almanac.
```

Optional suggestion - this may be slightly clearer.

Reply:
Love it, thanks!